tools/osx-pkg/osx-pkg.pkgproj

```diff
@@ -0,0 +1,2750 @@
+										<dict>
+											<key>COMPARATOR</key>
+											<integer>1</integer>
+											<key>OBJECT</key>
+											<integer>1</integer>
+											<key>UUID</key>
+											<string>41286F31-6755-4025-A7D1-04F7055B2AC8</string>
+										</dict>
+									</dict>
+									<key>HIDDEN</key>
+									<false/>
+									<key>STATE</key>
+									<integer>3</integer>
+								</dict>
+								<key>PACKAGE_UUID</key>
+								<string>134AB657-817C-4E65-AB53-BD59A5E272EB</string>
+								<key>REQUIREMENTS</key>
+								<array/>
+								<key>TITLE</key>
+								<array/>
+								<key>TOOLTIP</key>
+								<array/>
+								<key>TYPE</key>
+								<integer>0</integer>
+								<key>UUID</key>
+								<string>DA5CC0A2-F653-45B7-B5D6-BD0394BDCA8C</string>
+							</dict>
+							<dict>
+								<key>DESCRIPTION</key>
+								<array/>
+								<key>OPTIONS</key>
+								<dict>
+									<key>DEPENDENCY</key>
+									<dict>
+										<key>ENABLED_DEPENDENCY</key>
+										<dict>
+											<key>COMPARATOR</key>
+											<integer>1</integer>
+											<key>OBJECT</key>
+											<integer>1</integer>
+											<key>UUID</key>
+											<string>41286F31-6755-4025-A7D1-04F7055B2AC8</string>
+										</dict>
+										<key>ENABLED_MODE</key>
+										<integer>2</integer>
+										<key>SELECTED_DEPENDENCY</key>
+										<dict>
+											<key>COMPARATOR</key>
+											<integer>1</integer>
+											<key>OBJECT</key>
+											<integer>1</integer>
+											<key>UUID</key>
+											<string>41286F31-6755-4025-A7D1-04F7055B2AC8</string>
+										</dict>
+									</dict>
+									<key>HIDDEN</key>
+									<false/>
+									<key>STATE</key>
+									<integer>3</integer>
+								</dict>
+								<key>PACKAGE_UUID</key>
+								<string>2D16FD51-8800-422D-9E9B-EC59D2157FE8</string>
+								<key>REQUIREMENTS</key>
+								<array/>
+								<key>TITLE</key>
+								<array/>
+								<key>TOOLTIP</key>
+								<array/>
+								<key>TYPE</key>
+								<integer>0</integer>
+								<key>UUID</key>
+								<string>E6975970-F10F-41C5-8FD8-8D816D281D47</string>
+							</dict>
+							<dict>
+								<key>DESCRIPTION</key>
+								<array>
+									<dict>
+										<key>LANGUAGE</key>
+										<string>English</string>
+										<key>VALUE</key>
+										<string></string>
+									</dict>
+								</array>
+								<key>OPTIONS</key>
+								<dict>
+									<key>HIDDEN</key>
+									<false/>
+									<key>STATE</key>
+									<integer>2</integer>
+								</dict>
+								<key>PACKAGE_UUID</key>
+								<string>DC223951-DCCA-426C-8A88-B8F4A53FE9C6</string>
+								<key>REQUIREMENTS</key>
+								<array/>
+								<key>TITLE</key>
+								<array>
+									<dict>
+										<key>LANGUAGE</key>
+										<string>English</string>
+										<key>VALUE</key>
+										<string>uninstall io.js</string>
+									</dict>
+								</array>
+								<key>TOOLTIP<
```

Comment:
Maybe for consistency do `s/\t/  /g` here?  Two space indent is what's used everywhere else.